c.html.markdown

```diff
@@ -741,6 +749,7 @@ typedef void (*my_fnp_type)(char *);
 // Order of Evaluation
 ///////////////////////////////////////
 
+// From top to bottom, top is has higher precedence
```

Comment:
A mistake here?

Reply:
Oh you're right, "top has". I'll update.